assets/components/contributionsThankYou/contributionsThankYouPage.jsx

```diff
@@ -73,18 +90,27 @@ function RegularContributionsThankYouPage(props: PropTypes) {
         </PageSection>
       </div>
     </div>
-  ) : (
-    <EmailConfirmation />
   );
+}
+
+function getBodyCopy(props: PropTypes) {
+  if (props.contributionType === 'ONE_OFF') {
+    return '';
+  } else if (props.directDebitFields && props.directDebitFields.isDirectDebit) {
+    return DirectDebitDetails(props.directDebitFields);
```

Comment:
Not sure if this will work but:
`<DirectDebitDetails {...props.directDebitFields} />`
would be following the same pattern as with `<EmailConfirmation />`